distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -72,6 +72,12 @@ if [ ! -z "$CONF_FILE" ]; then
     exit 1
 fi
 
+# ES_USER and ES_GROUP settings were removed
+if [ ! -z "$ES_USER" ] || [ ! -z "$ES_GROUP" ]; then
+    echo "ES_USER and ES_GROUP settings are no longer supported. To run as a custom user/group use the .tar.gz distribution of Elasticsearch."
```

Comment:
I think you can say "archive distribution" as the zip distribution works in addition to the .tar.gz distribution.